spec.yaml

```diff
@@ -326,6 +326,24 @@ paths:
           content:
             application/json:
               example: "Awesome New Experiment Name"
+    delete: 
+      tags:
+        - Experiments
+      summary: Delete experiment
+      description: Support both git and MLflow experiments
+      operationId: deleteExperiment
+      parameters: 
+      - $ref: './components/parameters/Owner.yaml'
+      - $ref: './components/parameters/Repository.yaml'
+      - $ref: './components/parameters/ExperimentKey.yaml'
+      responses: 
+        '200':
+          description: Deleted experiment successfully
+        '404':
+          description: Experiment not found
+        '500':
+          description: Internal server error. Please reach out on our Discord server.
+
```

Comment:
I wouldn't write this as an explicit return code to be honest. 
There's only one other place in the spec where we return a 500